datasets/historical/eia_861_annual_energy_use_state_sector/dimension_mappings/scenario_to_scenario.csv

```diff
@@ -1,2 +1,2 @@
 from_id,to_id
-reference,decarb_high
+refrence,decarb_high
```

Comment:
I think this file be deleted now that you aligned this dataset's scenario with the project?